indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -19,11 +18,20 @@
                             'conflicting': conflicting_msg,
                             'negative': _("No reviewer proposed to accept this abstract"),
                             'mixed': _("Not all reviewers proposed to accept this abstract")} %}
+    {% set mapping_icon = {'positive': 'icon-checkmark',
+                           'conflicting': 'icon-checkmark',
+                           'negative': 'icon-cross',
+                           'mixed': 'icon-tilde',
+                           'in_progress': 'icon-circle-small',
+                           'not_started': 'icon-circle-small'} %}
```

Comment:
Not sure if any simplification won't end up just moving the complexity somewhere else. I don't think it's so ugly right now.